content/docs/integrations/broker/index.md

```diff
@@ -27,104 +27,83 @@ The Broker feature is available on our Growth Plan - see pricing page for more i
 
 This way Roadie does not need to have knowledge or access of your infrastructure endpoints nor authorization tokens. 
 
-
-
 ## Configuration
-
 ### Enabling broker connection
-
-
 The broker server instance is not enabled by default on tenants and thus needs to be requested to be enabled.
 
-Once enabled the broker server is exposed in a URL https://<your-tenant>.broker.roadie.so.
-The broker endpoint is secured by allow listing IP addresses. To set up broker connections you need to provide IP ranges where your broker clients are hosted so those can be configured to come through.
-
-
-### Configuring Broker client
+You can visit the settings page fro the broker at the following url in the application. `https://<your-tenant>.roadie.so/administration/settings/integrations/broker` 
 
-#### Broker client application
+The broker endpoint server is secured by allow listing IP addresses. You can configure the list of IP addresses in the settings page mentioned above.
 
-Broker client is a Node.js application which can be deployed as a Docker container into internal infrastructure. Roadie can provide Dockerfiles to construct broker client application by request, depending on the plugin. Alternatively Roadie can provide a base broker image Dockerfile which can be reused and enhanced with multiple different configuration files. 
+You can also test the client connections mentioned below from the settings page.
 
+### Configuring Broker client
 
-**Running the broker client application**
+There are three ways to run the broker client application:
 
+- Run one of the pre configured docker containers provided by roadie
+- Run the broker client nodejs application
+- Build a custom docker container
 
-The broker client can be installed and run as a standard node.js service. You can install the broker client when you have node.js and npm installed with a command `npm install --global snyk-broker`. After the global broker application is installed, you can run it with a command `broker --verbose` with verbose logging.
+#### Roadie Docker Containers
+Roadie provides pre configured docker containers containing pre-configured snyk-broker clients for use with backstage. At time of writing, we are providing a container image for kubernetes and sonarqube.
 
-Alternatively you can deploy the broker application as a container. Below you can find a base Dockerfile that can be used to construct the broker client image. 
+You can configure them with environment variables containing authentication credentials for the brokered service as well as the broker endpoint and token.
```

Comment:
```suggestion
You can configure them with environment variables containing authentication credentials for the brokered service as well as the broker endpoint and possible tokens.
```

Reply:
K8S_TOKEN && OTHER_SECURE_TOKEN etc. There's usually at least broker + the service token

Reply:
Take a look now. It had already referered to "authentication credentials for the the brokered service".

This could include tokens or passwords.

Let me know what you think.

Reply:
:+1: